### PR TITLE
Revert balance offset workaround

### LINKED ISSUE #4660: Revert balance offset workaround
### Problem

Due to a regression in services with a balance file not respecting its consensus timestamp invariant, mirror node had to implement a workaround to compensate. Services resolves the issue in 0.30 which is going to mainnet soon. 

### Solution

* Add balance file offsets to errata migration file and bump its checksum
* Remove 53 ns workaround in reconciliation job
* Remove 53 ns workaround in Rosetta 

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR reverts the 53ns account balance file offset workaround

- Add balance file offsets to errata migration file and bump its checksum
- Remove the 53ns workaround in reconciliation job
- Remove the 53ns workaround in Rosetta
 
**Related issue(s)**:

Fixes #4660

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/ErrataMigration.java [hunk 1]
```diff
@@ -131,11 +143,23 @@ protected void doMigrate() throws IOException {
...
-                "where consensus_timestamp in (:timestamps) and time_offset <> -1";
+        // Adjusts the balance file's consensus timestamp by -1 for use when querying transfers.
+        String sql = """
+                update account_balance_file set time_offset = -1
+                where consensus_timestamp in (:timestamps) and time_offset <> -1
+                """;
         int count = jdbcOperations.update(sql, new MapSqlParameterSource("timestamps", getTimestamps()));
+
+        // Set the fixed time offset for account balance files in the applicable range
+        sql = """
```
Comment (steven-sheehy) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/ErrataMigration.java:155]: nit: Would prefer variables not be reused. Should probably split separate SQL executions into separate methods.

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by edwin-greene — APPROVED** [review]
Looks good

**Review by MarcKriguerAtHedera — APPROVED** [review]
LGTM.

**Pull request merged**